browser/ai_chat/content_agent_tool_provider.cc

```diff
@@ -87,6 +92,11 @@ void ContentAgentToolProvider::StopAllTasks() {
   }
 }
 
+bool ContentAgentToolProvider::IsPausedByUser() {
+  // Tools shouldn't be attempted to run whilst actor task is paused by user
+  return actor_service_->GetTask(task_id_)->IsUnderUserControl();
```

Comment:
Do we need to check `task_id_->is_null()`? We do in some other places (like Stop)

Reply:
Although that also should never happen considering when we call StopTask we first create a new one 🤷